src/apps/fee-list/FeeList.tsx

```diff
@@ -51,54 +40,17 @@ const FeeList: FC = () => {
   );
 
   useEffect(() => {
-    if (fbsFees) {
-      const feesPrePaymentChange = getFeesInRelationToPaymentChangeDate(
-        fbsFees,
-        true
-      ).length;
-      if (feesPrePaymentChange > 0) {
-        setItemsPrePaymentChange(
-          getFeesInRelationToPaymentChangeDate(fbsFees, true)
-        );
-      }
-      const feesPostPaymentChange = getFeesInRelationToPaymentChangeDate(
-        fbsFees,
-        false
-      ).length;
-      if (feesPostPaymentChange > 0) {
-        setItemsPostPaymentChange(
-          getFeesInRelationToPaymentChangeDate(fbsFees, false)
-        );
-      }
-    }
-  }, [fbsFees]);
-
-  useEffect(() => {
-    if (totalFeePrePaymentChange > 0) {
-      return;
-    }
-    const totalFee = itemsPrePaymentChange?.reduce(
-      (accumulator, { amount }) => accumulator + amount,
-      0
-    );
-    if (totalFee) {
-      setTotalFeePrePaymentChange(totalFee);
-    }
-  }, [itemsPrePaymentChange, totalFeePrePaymentChange]);
-
-  useEffect(() => {
-    if (totalFeePostPaymentChange > 0) {
```

Comment:
You have a lot of functionality being removed here. I think it would be worth to write in the commit message why it deletion was needed.